### PR TITLE
Login page

### LINKED ISSUE #22: Login page
The project also needs a corresponding login page to sign-in users

### PR DESCRIPTION
**Pull request by sreejithvs333**
Fix #22 

Hi @micaiah-effiong 

I've served the pages as static files using express

Links
http://localhost:3000/static/index.html
http://localhost:3000/static/login.html

In app.js you can see the code for the same
`//Serving static files`
`app.use('/static', express.static('public'))`

Please see the designs below

**Desktop View**

Screenshot 2020-10-04 at 9 25 09 AM

**Mobile View**
Screenshot 2020-10-04 at 9 32 27 AM

Hope this helps!

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, "public")));
+app.use('/static', express.static('public'))
```
Comment (micaiah-effiong) [inline app.js]: You're serving statics files over the ```/static``` route, your but the HTML does not use the  ```/static``` route to fetch the files....
This change will mean that all statics files should served over ```/static```

Revert to the root route

File: app.js [hunk 2]
```diff
@@ -36,4 +36,7 @@ app.use(function (err, req, res, next) {
   res.json({ success: false, err, msg: err.message });
 });
 
+//Serving static files
+app.use('/static', express.static('public'))
```
Comment (micaiah-effiong) [inline app.js]: This line is a duplicate of the changes in line 20

**Review by micaiah-effiong — CHANGES_REQUESTED** [review]
Revert ```line 20``` to the root route
Remove ```line 49``` it's a duplicate

File: app.js (hunk 2 above)
Comment (sreejithvs333) [reply app.js]: Removed static file serving in app.js

**Comment by micaiah-effiong** [comment]
Just add a static middleware so the static files can be served...
You had mistakenly removed everything

app.use(express.static(path.join(__dirname, "public")));

File: app.js [hunk 3]
```diff
@@ -17,7 +17,6 @@ app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
```
Comment (micaiah-effiong) [inline app.js:19]: Just add a static middleware so the static files can be served...
You had mistakenly removed everything

```app.use(express.static(path.join(__dirname, "public")));```

**Review by micaiah-effiong — CHANGES_REQUESTED** [review]
Make the changes 
add a static middleware
```app.use(express.static(path.join(__dirname, "public")));```

File: app.js (hunk 3 above)
Comment (sreejithvs333) [reply app.js:19]: Done 👍

**Review by micaiah-effiong — APPROVED** [review]
Good work...

**Comment by micaiah-effiong** [comment]
@sreejithvs333 feel to start the repo, and also keep an eye for issues you could help out with....

**Pull request merged**